From 661068853210ea3a9d3335f1117d912972723f72 Mon Sep 17 00:00:00 2001
From: Christophe Fergeau <cfergeau@redhat.com>
Date: Tue, 9 Jan 2018 14:42:25 +0100
Subject: [PATCH] Mark PrintScreen as translatable
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

It's for example ImprÉcran on French keyboards.

Signed-off-by: Christophe Fergeau <cfergeau@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1510411
Fabiano Fidêncio <fidencio@redhat.com>

(cherry picked from commit da336ab4b3e4ce1d172b53d43a516ed5d3506a1f)
---
 src/virt-viewer-window.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/virt-viewer-window.c b/src/virt-viewer-window.c
index a796685..a05140b 100644
--- a/src/virt-viewer-window.c
+++ b/src/virt-viewer-window.c
@@ -555,7 +555,7 @@ static const struct keyComboDef keyCombos[] = {
     { { GDK_KEY_Control_L, GDK_KEY_Alt_L, GDK_KEY_F11, GDK_KEY_VoidSymbol }, N_("Ctrl+Alt+F11"), NULL},
     { { GDK_KEY_Control_L, GDK_KEY_Alt_L, GDK_KEY_F12, GDK_KEY_VoidSymbol }, N_("Ctrl+Alt+F12"), NULL},
     { { GDK_KEY_VoidSymbol }, "" , NULL},
-    { { GDK_KEY_Print, GDK_KEY_VoidSymbol }, "_PrintScreen", NULL},
+    { { GDK_KEY_Print, GDK_KEY_VoidSymbol }, N_("_PrintScreen"), NULL},
 };
 
 static guint
